hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/TestRMHAForAsyncScheduler.java

```diff
@@ -87,6 +88,7 @@ public void testAsyncScheduleThreadStateAfterRMHATransit() throws Exception {
     startRMs();
     // register NM
     MockNM nm = rm1.registerNode("192.1.1.1:1234", 8192, 8);
+    CapacitySchedulerTestUtilities.getCapacityScheduler(rm1, 8,8);
```

Comment:
The parameters 8, 8 seem arbitrary, especially because the line above uses MBs - can you please introduce variables and use the same prefix?

Reply:
done